src/app/DataViews.jsx

```diff
@@ -97,35 +96,28 @@ const DataViewsContent = () => {
           <DataList
             {...{
```

Comment:
Is it necessary to spread the object like that? Couldn't it be written as:

    {
        data,
        idKey,
        onSelect,
        hasMoreData,